recipes/roocs-utils/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "roocs-utils" %}
+{% set package_name = "roocs_utils" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ package_name }}-{{ version }}.tar.gz
+  sha256: e44ca4c8c66c283aec96f36a2be810fa9f599438ee00dee0a94e4d6001721709
+
+build:
+  #  noarch: python
+  skip: True  # [win]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
Try [grayskull](https://github.com/conda-incubator/grayskull) next time you submit a recipe or update this one. It will get the recipe in the "latest" format, like removing the unnecessary quotes here.